parfiles/periodic_BC_test7.prm

```diff
@@ -0,0 +1,103 @@
+# based on boundary_velocity_function_spherical.prm
+# testing for periodic conditions in phi 
+# (direction in spherical_shell.cc and core.cc currently set to 1
+#  0=r, 1=phi)
+# renaming of X periodic to Phi periodic for periodic condition
+#  in spherical shell
+
+
+set Dimension                              = 2
+
+set Use years in output instead of seconds = true
+set End time                               = 1e6
+
+set Adiabatic surface temperature          = 1613.0
+set Output directory                       = outfiles/periodic_BC_test7
+
+subsection Geometry model
+  set Model name = spherical shell
+
+  subsection Spherical shell
+    set Opening angle = 90
+    set Inner radius = 3481000
+    set Outer radius = 6336000
+    set Phi periodic = true
```

Comment:
I don't understand how this works. Do you want to keep all these files?

Reply:
I think we should just keep this latest parameter file periodic_BC_test7.prm (with a new name?). It's much easier to tell if the periodicity is working or not with this case, compared to the first one I was using periodic_BC_test1.prm.

Reply:
Sounds good. Can you then please remove all of the other files, and move this one either into `tests/` or `cookbooks/periodic_boundaries`? We can discuss tomorrow if this should become a test or a cookbook. In case of a cookbook we may be able to coordinate with #3792 and @Geoniette on a combined cookbook for box and spherical boundaries, but it would require writing a page or two for the manual about periodic boundaries, so this may be more work.
Also lets discuss in our meeting tomorrow how to move the changes over into the geometry model.